.pagerduty
  h1 PagerDuty
  if pager_duty.error
    p The pager_duty API returned the following error:
    p.alert!=trace(pager_duty.error)
  else
    h4 Currently On Call:
    p
    ul
      li
        strong  #{pager_duty.data.entries[0].user.name} <br>
        | from: #{pager_duty.data.entries[0].start}<br>
        |   to: #{pager_duty.data.entries[0].end}
    p
    if pager_duty.data.entries[1]
      h4 Next On Call:
      p
      ul
        li
          strong  #{pager_duty.data.entries[1].user.name} <br>
          | from: #{pager_duty.data.entries[1].start}<br>
          | to: #{pager_duty.data.entries[1].end}
